src/connectedk8s/azext_connectedk8s/_troubleshootutils.py

```diff
@@ -16,6 +16,7 @@
 from knack.log import get_logger
 from azure.cli.core import telemetry
 import azext_connectedk8s._constants as consts
+import azext_connectedk8s._utils as azext_utils
```

Comment:
import it as "import azext_connectedk8s._utils as utils" for consistency with other files